src/NEAR/Serialization.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Serialization.h"
+
+#include "../NEAR/Address.h"
+#include "../Base64.h"
+#include <TrustWalletCore/TWHRP.h>
+
+using namespace TW;
+using namespace TW::NEAR;
+using namespace TW::NEAR::Proto;
+
+
+static void writeU8(Data& data, uint8_t number) {
```

Comment:
can you reuse code in `BinaryCoding.h`?

Reply:
yes, thanks for suggestion!

I'll delegate to `BinaryCoding` where there are matching functions